Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astral-tokio-tar"
-version = "0.5.0"
+version = "0.5.0-rc.1"
```

Comment:
Why `-rc.1`?

Reply:
Transplanting comment:

I only bother with prereleases when i need to publish it for testing and there's existing users I don't want to "upgrade" to that version (since cargo makes it an opt-in to ever see prereleases).

I don't see any reason to use that here.

Reply:
I usually start with a prerelease for new projects so a failed release doesn't get the git tag.